### DISCUSSION TITLE
Banner Error undefined property IronSourceBannerSize

### DISCUSSION THREAD

**Discussion by jeuxdaily** [distriqt/ANE-IronSource#42] [Q&A]
Hi
I hope that you fine
I'm using distriqt/ANE-IronSource AIR sdk 33.1.1.935 the interrstitial work good
but when I Add The banner code

IronSource.instance.loadBanner( 
    IronSourceBannerSize.BANNER,
    IronSource.POSITION_BOTTOM
);
it's get this error:
Access of undefined property IronSourceBannerSize

**Comment by marchbold** [comment]
Did you add the import for the class? 

```as3
import com.distriqt.extension.ironsource.IronSourceBannerSize;
```

**Comment by jeuxdaily** [comment]
thanks for your answer
I get Another erreur
1119: Access of possibly undefined property BANNER through a reference with static type Class.
IronSource.instance.init( IRONSRC_APP_KEY, [ IronSource.BANNER ] );

**Reply by marchbold** [reply]
Are you sure you are using the distriqt IronSource class and not the one provided by IronSource? Did you include the import for the IronSource class as below:

```as3
import com.distriqt.extension.ironsource.IronSource;
```

**Comment by jeuxdaily** [comment]
yes I do

**Reply by marchbold** [reply]
Are you sure you are using the distriqt IronSource class and not the one provided by IronSource?

**Reply by jeuxdaily** [reply]
yes sure

this extension

    <extensionID>androidx.core</extensionID>
    <extensionID>com.distriqt.Core</extensionID>
    <extensionID>com.distriqt.IronSource</extensionID>
    <extensionID>com.distriqt.playservices.Ads</extensionID>
    <extensionID>com.distriqt.playservices.AdsIdentifier</extensionID>
    <extensionID>com.distriqt.playservices.Base</extensionID>
    <extensionID>com.jetbrains.kotlin</extensionID>

and this Importaion

		import com.distriqt.extension.ironsource.events.RewardedVideoAdEvent;
		import com.distriqt.extension.ironsource.IronSourceBannerSize;
		import com.distriqt.extension.ironsource.IronSource;

and this code interstitial and Rewadvideo work good

IronSource.instance.init( IRONSRC_APP_KEY, [ IronSource.INTERSTITIAL ] );
IronSource.instance.init( IRONSRC_APP_KEY, [ IronSource.REWARDED_VIDEO ] );

but Only code BANNER get error

IronSource.instance.init( IRONSRC_APP_KEY, [ IronSource.BANNER ] );

**Comment by jeuxdaily** [comment]
so 

Well when I write the code 
IronSource.instance.init( IRONSRC_APP_KEY, [ IronSource.

Well when I write the code

A small window appears with all the possible codes

![image](https://user-images.githubusercontent.com/14032996/200622027-8a88d853-e1d5-4554-92ce-cdb1a2169e0f.png)

that mean BANNER not exit

**Reply by marchbold** [reply]
What IDE are you using? The BANNER constant definitely is defined in the ANE. 

You can just replace it with `"banner"` if you wish.

**Comment by jeuxdaily** [comment]
thanks for your help
Now It's working

**Reply by jeuxdaily** [reply]
I change BANNER with IronSourceBannerSize.BANNER and it's working fine
